src/components/InventoryGroups/Modals/SearchInput.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import {
+    HelperText,
+    HelperTextItem,
+    Select,
+    SelectOption
+} from '@patternfly/react-core';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import debounce from 'lodash/debounce';
+
+const SearchInput = () => {
```

Comment:
This component makes some really multiple and ineffecient computations. I can see with profiler that the component and the list is re-rendered too many times (has constantly brown edges). Also the behavior is with the huge delays (it's well seen on the video)

[Screencast from 03-29-2023 11:23:18 AM.webm](https://user-images.githubusercontent.com/31385370/228489671-ce4fe25f-3d2e-4d58-be8a-fdd703ee8710.webm)